modules/virt-configuring-disk-sharing-lun-web.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-configuring-shared-volumes-for-vms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-configuring-disk-sharing-lun-web_{context}"]
+= Configuring disk sharing by using LUN and the web console
+
+You can use the {product-title} web console to:
+
+* Configure disk sharing by using LUN
+
+* Enable the `persistentreservation` feature gate
+
+.Procedure
+
+. Navigate to *Virtualization* -> *VirtualMachines* in the web console.
+
+. Select a VM to open the *VirtualMachine details* page.
+
+. Expand *Storage*.
+
+. On the *Disks* tab, click *Add disk*.
+
+. Specify the *Name*, *Source*, *Size*, *Interface*, and *Storage Class*.
+
+. Select *LUN* as the *Type*.
+
+. Select the *Apply optimized StorageProfile settings* checkbox.
+
+. Select *Shared access (RWX)* as the *Access Mode*.
+
+. Select *Block* as the *Volume Mode*.
```

Comment:
this is not necessary for the purpose of shared disk

Reply:
When I documented this feature [for the command line](https://github.com/openshift/openshift-docs/pull/64333), it was necessary to mention both the **Shared access** and **block** as the **Volume Mode** requirements.  The **Apply optimized StorageProfile settings** checkbox is selected in the screenshots shown in the [design spec](https://docs.google.com/document/d/1_fbw2BUZFRmArjhdKc32lTEqU9MRHsj2KYBxbR6ZL_0/edit#heading=h.toz5r6ewcnq6). @yfrimanm, was that not what you meant in the design spec?